include/xtensor/xstrides.hpp

```diff
@@ -14,15 +14,11 @@
 #include <numeric>
 
 #include "xexception.hpp"
+#include "xlayout.hpp"
+#include "xtensor_forward.hpp"
 
```

Comment:
If you define the layout enum in `xtensor_forward.hpp` I think you will entually keep the include here.